dockerbuilder/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
```

Comment:
12.04 is the earliest LTS we support - so why pick this arbitrary version that is neither that nor the latest?